lucille-core/src/main/java/com/kmwllc/lucille/stage/DropDocument.java

```diff
@@ -0,0 +1,22 @@
+package com.kmwllc.lucille.stage;
+
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.typesafe.config.Config;
+import java.util.Iterator;
+
+public class DropDocument extends Stage {
```

Comment:
please add javadoc that explains what this stage is for
mention that this stage is intended to be used with conditions